app/models/item.rb

```diff
@@ -11,4 +11,13 @@ class Item < ApplicationRecord
 
   enum status: { "Enabled" => 0, 
   "Disabled" => 1 }
+
+  def format_inv_date(invoice_id)
```

Comment:
Not sure where the issue is exactly, but when I open up some merchant dashboard pages in the rails server, it doesn't seem to be ordering correctly.

<img width="479" alt="Screenshot 2024-04-17 at 7 41 49 AM" src="https://github.com/mel-langhoff/little-shop-7/assets/155924313/0a8373c7-7bb7-426a-a9ea-fb7175fa886d">